### PR TITLE
Send Angular Version as part of auth0Client header

### PR DESCRIPTION
**Pull request by frederikprijck**
This PR sends the version of Angular being used as part of the auth0Client.

### CODE REVIEW

File: projects/auth0-angular/src/lib/auth.client.ts [hunk 1]
```diff
@@ -23,7 +23,10 @@ export class Auth0ClientFactory {
       auth0Client: {
         name: useragent.name,
         version: useragent.version,
-      },
+        env: {
+          'angular/core': VERSION.full
+        }
+      } as any,
```
Comment (frederikprijck) [inline projects/auth0-angular/src/lib/auth.client.ts:29]: Will need to drop this once https://github.com/auth0/auth0-spa-js/pull/913 is merged

Comment (Widcket) [reply projects/auth0-angular/src/lib/auth.client.ts:29]: It has been merged now.

**Review by stevehobbsdev — APPROVED** [review]

File: projects/auth0-angular/src/lib/auth.client.ts (hunk 1 above)
Comment (frederikprijck) [reply projects/auth0-angular/src/lib/auth.client.ts:29]: Thanks. I forgot the fact that it will also need a release. So will hold of merging here.

Comment (frederikprijck) [reply projects/auth0-angular/src/lib/auth.client.ts:29]: Or maybe we should merge as is for now. As it should work, and might be a bit weird to cut a release in SPA-JS only for this.

**Pull request merged**